Svc/ActiveLogger/docs/sdd.md

```diff
@@ -1,23 +1,20 @@
-\page SvcActiveLoggerComponent Svc::ActiveLogger Component
 # Svc::ActiveLogger Component
 
 ## 1. Introduction
 
-The `Svc::ActiveLogger` component processes events from other components. The events are put in packets and sent to an external user like a ground system. The component provides event filtering and writing event logs to a file.
+The `Svc::ActiveLogger` component processes events from other components. The events are put in packets and sent to an
+external component like the ground interface. The component provides event filtering capability such that events may
+be turned off via ID or severity.
 
 ## 2. Requirements
 
 The requirements for `Svc::ActiveLogger` are as follows:
 
 Requirement | Description | Verification Method
 ----------- | ----------- | -------------------
-AL-001 | The `Svc::ActiveLogger` component shall store events. | Inspection; Unit Test
-AL-002 | The `Svc::ActiveLogger` component shall store a configurable circular buffer of events by severity. | Unit Test
-AL-003 | The `Svc::ActiveLogger` component shall write the circular buffers to a file upon command | Unit Test
-AL-004 | The `Svc::ActiveLogger` component shall have commands to filter events based on event severity. | Unit Test
-AL-005 | The `Svc::ActiveLogger` component shall filter events on receipt of the event on the caller's thread but before being placed in the event buffer. | Unit Test
-AL-006 | The `Svc::ActiveLogger` component shall filter events on the thread of the component before sending them as packets, but after they have been stored in the circular buffers | Unit Test
-AL-007 | The `Svc::ActiveLogger` component shall filter events based on the event ID. | Unit Test 
+AL-001 | The `Svc::ActiveLogger` component shall queue events and compose them into downlink packets. | Inspection; Unit Test
+AL-002 | The `Svc::ActiveLogger` component shall have commands to filter events based on event severity. | Unit Test
+AL-003 | The `Svc::ActiveLogger` component shall have commands to filter events based on the event ID. | Unit Test 
```

Comment:
Is there no requirement for sending out fatal events on the FatalAnnounce port?
I guess it wasn't there before, so this isn't a blocker to me.